src/index.tsx

```diff
@@ -7,14 +7,19 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { App } from "./Components/App/App";
 import * as serviceWorker from "./serviceWorker";
-
+import { FeatureToggleClient } from "FeatureToggleClient";
+import { AuthorizationClient } from "@itwinjs-sandbox";
 import "./index.scss";
 import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
 
-ReactDOM.render(
-  <App />,
-  document.getElementById("root"),
-);
+(async () => {
+  await FeatureToggleClient.initialize();
```

Comment:
Should be moved downstream before it's usage. Adds ~150ms to app statup delay. Not sure if feature toggling is really needed at all. Can anyone comment?

Reply:
Move Feature Toggle client init to Tab Nav (it's the only place that it is getting used).